.github/workflows/pythonapp.yml

```
@@ -37,3 +37,12 @@ jobs:
       run: |
         pip install pytest
         python -m pytest
+    - name: Run App
+      run: |
+        streamlit run app.py &
```

@garysieling update per #169 to

```suggestion
        streamlit run src/app.py &
```